src/parseTimeoutSeconds.ts

```diff
@@ -0,0 +1,26 @@
+export const defaultTimeout = 15 * 60
+
+export function parseTimeoutSeconds(timeoutString?: string): number {
```

Comment:
there should be a library to do this for you, instead of running your own implementation

Reply:
I couldn't find anything that matches the Scala's timeout format we use

Reply:
this:
https://github.com/josephtzeng/hocon-parser/blob/3baba010f08e4a25d08ca190053693d30ed74dc1/tests/resources/configs/include-from-list/test01.conf#L58
looks appropriate, isn't it?

Reply:
it has no typescript facades :(